packages/react-ui-core/src/List/__tests__/List-test.js

```diff
@@ -27,6 +50,24 @@ describe('List', () => {
       expect(wrapper.find('ListItem').at(0).prop('nodeType')).toEqual('span')
     })
 
+    it('passes through object props', () => {
+      const snap = renderer
+        .create(<List items={menuItems} />)
+        .toJSON()
+      expect(snap).toMatchSnapshot()
+    })
+
+    it('handles labelless items', () => {
+      const errorFunc = global.console.error
+      global.console = { error: jest.fn() }
```

Comment:
will this wipe out console for all other tests?

Reply:
No, because it gets reset after [here](https://github.com/jest-community/eslint-plugin-jest#usage).  We couldn't find an easier way to do this.  Open to suggestions.

Reply:
yeah but you're taking out console.log and otheres - what about 

```
  let original

  beforeEach(() => {
    original = console.error
    console.error = jest.fn()
    console.error('you cant see me')
  })

  afterEach(() => {
    console.log('log still works')
    console.error('you cant see me')
    console.error.mockClear()
    console.error = original
    console.error('now you can')
  })

```

Reply:
We don't need to before each them... I see the issues though, it should probably be:
```
global.console.error = jest.fn()
```